Musigwa/RNLevelUp
# #166258436 Design the Profile Screen of the Application

#### What does this PR do?

- It finishes the profile screen design and functionality
#### Description of Task to be completed?

- Design the profile screen and consume the API to get a single developer's profile info
- Implement `share` functionalities onto the profile screen so, the user will be able to share via different platforms

#### How should this be manually tested?
- After cloning the repository, check out `ft-design-the-profile-screen-166258436` branch
- Checkout the `README.md` file for set up instructions
- Run `npm install` to install the project's dependencies
- Run `npm test` to runs all test scenarios and see the results in from the terminal
- After all, run `npm start` to open the application in `ios simulation` or `android emulator` or just a physical device. 
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #166258436
#### Screenshots (if appropriate)

- Android device
1. Profile screen
<img width="330" alt="Screen Shot 2019-06-07 at 10 39 56" src="https://user-images.githubusercontent.com/29999424/59092208-45cca680-8911-11e9-9274-0cc00fc1298c.png">

2. Share profile modal
<img width="327" alt="Screen Shot 2019-06-07 at 10 42 51" src="https://user-images.githubusercontent.com/29999424/59092231-54b35900-8911-11e9-849c-f3cc1c128f56.png">

3. GitHub profile web-view 
<img width="329" alt="Screen Shot 2019-06-07 at 10 41 32" src="https://user-images.githubusercontent.com/29999424/59092257-63017500-8911-11e9-8771-f6bfec648907.png">

- IOS device
1. Profile screen
<img width="348" alt="Screen Shot 2019-06-07 at 10 40 22" src="https://user-images.githubusercontent.com/29999424/59092621-516c9d00-8912-11e9-8a37-62361aa5fe75.png">

2. Share profile modal
<img width="349" alt="Screen Shot 2019-06-07 at 10 42 12" src="https://user-images.githubusercontent.com/29999424/59092630-592c4180-8912-11e9-8f5d-b0755927c3ee.png">

3. GitHub profile web-view 
<img width="349" alt="Screen Shot 2019-06-07 at 10 40 57" src="https://user-images.githubusercontent.com/29999424/59092637-5f222280-8912-11e9-8ca5-4b72cfa28592.png">

#### Questions:
- None
